### ISSUE TITLE
Focus management after closing overlays [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#193]
Closing overlays does not properly return focus back to the the button that is responsible for triggering the overlay on the previous screen, instead focus moves back to the top of the screen. The one exception is the “close” button found on the bottom right corner of the last (step 4) “How it works” screen. Focus is managed properly when closing the overlay with that specific button.

**Comment by ckiss** [comment]
fixed, avail in next build

**Comment by willslone** [comment]
I retested this and it is fixed in some cases and not others. For example, closing the Gender identity overlay by using the close graphical button does not return focus to the control that opened the overlay. Closing the Gender identity overlay by selecting an option in the overlay does return focus properly.

**Comment by helloantoine** [comment]
@floridemai Can you have a look and make sure the fix is applied to every overlays please?

**Comment by JohnAGill** [comment]
These should have all been fixed now and be in the next build
